packages/stark-ui/src/modules/app-sidebar/services/app-sidebar.service.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { StarkAppSidebarService } from "./app-sidebar.service";
+import { StarkAppSidebarOpenEvent } from "./app-sidebar-open-event.intf";
+import createSpy = jasmine.createSpy;
+
+describe("ValueService", () => {
+	let service: StarkAppSidebarService;
+	beforeEach(() => {
+		service = new StarkAppSidebarService();
+	});
+
+	it("openMenu() should raise the correct event", () => {
+		service.openSidebar$.subscribe((event: StarkAppSidebarOpenEvent) => {
+			expect(event).toEqual({ mode: "menu", sidebar: "left" });
+		});
+		service.openMenu();
+	});
+
+	it("openLeft() should raise the correct event", () => {
+		service.openSidebar$.subscribe((event: StarkAppSidebarOpenEvent) => {
+			expect(event).toEqual({ mode: "regular", sidebar: "left" });
+		});
+		service.openLeft();
+	});
+
+	it("openRight() should raise the correct event", () => {
+		service.openSidebar$.subscribe((event: StarkAppSidebarOpenEvent) => {
+			expect(event).toEqual({ sidebar: "right" });
+		});
+		service.openRight();
+	});
+
+	it("close() should raise an event", () => {
+		const closeSidebars: jasmine.Spy = createSpy("closeSidebarsSpy");
+		service.closeSidebar$.subscribe(() => {
+			closeSidebars();
+		});
+		service.close();
+		expect(closeSidebars).toHaveBeenCalled();
```

Comment:
You can use the "toHaveBeenCalledTimes(1)" matcher to be more explicit and to be sure it was called only once